client/src/components/teams/SubTeams.vue

```diff
@@ -4,12 +4,7 @@
     <v-col cols="12">
       <v-row justify="space-between" no-gutters>
         <v-col cols="10">
-          <v-text-field
-            :rules="[state.rules.required]"
-            variant="outlined"
-            v-model="state.newSubTeamName"
-            label="Add subteam"
-          />
+          <v-text-field :rules="[rules.required]" variant="outlined" v-model="newSubTeamName" label="Add subteam" />
         </v-col>
         <v-col cols="2">
           <v-icon class="float-right" @click="addSubTeam" icon="mdi mdi-plus" size="48px" />
```

Comment:
I should see the textfield or plus button if im not the admin of the team. Theres also too much spacing between the textfield and button (make it similar to events page)